src/Features/ERDDAP/ForecastsMetadata/Menu/index.tsx

```diff
@@ -1,8 +1,8 @@
 /**
  * Dropdown menu for accessing forecasts via buoy barn
  */
-import * as React from "react"
-import { Link } from "react-router-dom"
+import Link from "next/link"
+import React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6288046)